insitu/templates/data_responsible/network/detail.html

```diff
@@ -54,8 +60,8 @@ <h1 class="col-sm-12">
         <tr class="row">
           <th class="col-md-4 col-sm-6">Members</th>
           <td class="col-md-8 col-sm-6">
-            {% for member in responsible.members.all %}
-                <a href="{% url 'responsible:detail' member.pk %}">{{ member.name }}</a>
+            {% for network in responsible.networks.all %}
```

Comment:
Why networks? You are displaying members, which means you want to see all data responsible that have this network in data_responsible.networks, not the responsible's networks.